core/vm/interface.go

```diff
@@ -23,26 +23,36 @@ import (
 	"github.com/ethereum/go-ethereum/core/types"
 )
 
+// Quorum uses a cut-down StateDB, MinimalApiState. We leave the methods in StateDB commented out so they'll produce a
```

Comment:
Is this design choice documented anywhere? is there a reason to not implement the full interface?